addons/l10n_tw_edi_ecpay/models/account_move.py

```diff
@@ -0,0 +1,280 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+import datetime
+import uuid
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.addons.l10n_tw_edi_ecpay.utils import EcPayAPI
```

Comment:
datetime is not used, also ordering :)

```suggestion
import base64
import uuid

from odoo import _, api, fields, models
from odoo.addons.l10n_tw_edi_ecpay.utils import EcPayAPI
from odoo.exceptions import UserError
```